### PR TITLE
Docs v2 setup

### PR DESCRIPTION
**Pull request by PaulLeCam**
Include docs in source comments to create expected output in docsite with TypeDoc pages generation.

### CODE REVIEW

File: docs/modules/framework.md [hunk 1]
```diff
@@ -1,18 +1,176 @@
 # Module: framework
 
-Framework APIs
+React framework for decentralized apps.
+
+## Purpose
+
+The `framework` module is the highest-level abstraction provided by the Self.ID SDK, aimed at
+helping developers quickly getting started with building decentralized apps using Ceramic with
```
Comment (zachferland) [inline docs/modules/framework.md:8]:
```suggestion
helping developers to quickly get started with building decentralized apps using Ceramic with
```

File: docs/modules/framework.md [hunk 2]
```diff
@@ -1,18 +1,176 @@
...
+React.
+
+The framework is built on top of the [`core`](core.md), [`web`](web.md), [`react`](react.md),
+[`ui`](ui.md) and [`multiauth`](multiauth.md) modules to provides APIs and UI components to easily
```
Comment (zachferland) [inline docs/modules/framework.md:12]:
```suggestion
[`ui`](ui.md) and [`multiauth`](multiauth.md) modules to provide APIs and UI components to easily
```

File: docs/modules/image_utils.md [hunk 3]
```diff
@@ -1,11 +1,57 @@
 # Module: image-utils
 
-Image utilities APIs
+Image utilities for Self.ID profiles.
+
+## Purpose
+
+The `image-utils` module provides various utility functions for images, such as selecting the
+most suitable format among sources, resizing and uploading, mostly meant to be used in a Web
```
Comment (zachferland) [inline docs/modules/image_utils.md:8]:
```suggestion
most suitable format among sources, resizing and uploading, and is mostly meant to be used in a Web
```

**Review by zachferland — APPROVED** [review]

**Comment by PaulLeCam** [comment]
Thanks for the suggestions!

**Pull request merged**